src/components/resources-section/resources-list-item/ResourcesListItem.module.scss

```diff
@@ -69,3 +69,7 @@
   gap: 1rem;
   text-align: center;
 }
+
+.small_number {
+  font-size: smaller;
```

Comment:
As the previous comment, the better way to work with responsive design is using rem or em.

Reply:
Same, thx!